parley/src/layout/data.rs

```diff
@@ -223,6 +227,7 @@ impl<B: Brush> Default for LayoutData<B> {
             text_len: 0,
             width: 0.,
             full_width: 0.,
+            content_widths: OnceCell::new(),
```

Comment:
It would be good to add the changes to the changelog, and mention as breaking change `Layout` is no longer `sync`.